### PR TITLE
Support Ice 3.7 and Ubuntu 20.04

### PR DESCRIPTION
**Pull request by tinyhaggis**
This pull request includes all changes and updates to MumPi to support newer operating systems such as Ubuntu 18.04 and 20.04.  Since older versions of Ubuntu are no longer supported, support for obsolete versions of Mumble below 1.3.0 have been removed.

### CODE REVIEW

**Comment by AtomicHaggis** [comment]
As requested, this PR has been squashed down before submission so it only contains two commits.

**Comment by AtomicHaggis** [comment]
I have tested this update with a clean installation of Ubuntu 20.04 and it operates correctly.

**Comment by Kissaki** [comment]
Awesome!

I will broadly look over the changes a bit later, but overall trust you on them and their functioning. :)

**Comment by AtomicHaggis** [comment]
Yeah, sorry for the delay in getting the PR submitted with the squashed commits.  Hopefully this revitalises the project and we can bring it up to date later with Mumble 1.4 once it goes stable.

File: classes/ServerInterface.php [hunk 1]
```diff
@@ -5,7 +5,17 @@
 require_once(MUMPHPI_MAINDIR.'/classes/HelperFunctions.php');
 require_once(MUMPHPI_MAINDIR.'/classes/MessageManager.php');
 
-if (extension_loaded('ice') && function_exists('Ice_intVersion') && Ice_intVersion() >= 30400) {
+if(!extension_loaded("ice"))
+{
+	printJson("IcePHP extension is not loaded. Revise your IcePHP installation.");
```
Comment (Kissaki) [inline classes/ServerInterface.php:10]: Is printJson a valid function? I don’t think so?

**Comment by AtomicHaggis** [comment]
That's correct.  AtomicHaggis.  That other one was using a work account.

File: classes/ServerInterface.php (hunk 1 above)
Comment (AtomicHaggis) [reply classes/ServerInterface.php]: My mistake.  I had a test function and that was a left-over command.  Shall I submit a PR with that corrected?

**Comment by AtomicHaggis** [comment]
Yep, no problem. I'll grab it at test it in the morning.

**Comment by AtomicHaggis** [comment]
Work's been pretty hectic.  I'll try to get to this tomorrow.

**Comment by Kissaki** [comment]
@AtomicHaggis Will you be able to test it soon(ish)? Otherwise I will merge it as-is.

**Comment by AtomicHaggis** [comment]
@Kissaki Life's been a bit chaotic!  I'll get to testing it this weekend.  I'll let you know on Sunday.

**Comment by Kissaki** [comment]
I am merging it as-is :)

and will tag a release version

**Pull request merged**